controller/volume_controller.go

```diff
@@ -2416,7 +2416,7 @@ func (c *VolumeController) getReplicaCountForAutoBalanceNode(v *longhorn.Volume,
 		}
 	}
 	if len(nodeExtraRs) == len(readyNodes) {
-		log.Info("Balanced, all ready nodes are used by this volume")
+		log.Debugf("Balanced, all ready nodes are used by this volume")
 		return 0, nodeExtraRs, nil
```

Comment:
How about the below code?

```go
	log.Infof("Found %v node available for auto-balance duplicates in %v", adjustCount, nodeExtraRs)

```

Reply:
This message is important scheduling decision message, need to keep in Info